metabase/metabase
# create unified BE path for isomorphic visualizations

### Description

First static visualizations that used javascript for rendering were incomplete duplicates which tried to reproduce the logic of the dynamic visualizations. The logic was spread between Clojure and Javascript. This approach has multiple downsides because all viz features have to be implemented twice and the static chart versions were far behind.

With the new [technical vision](https://www.notion.so/metabase/Viz-Revamp-Research-and-Planning-4198b2726ac64f1d89ad1de6187532d2) for visualizations we will be using the same code both for dynamic and static charts. This PR creates a single BE rendering path for isomorphic visualizations and removes code of the BE code of line/area/bar/combo static charts (FE will be removed further).

### How to verify

1. Create a dashboard with line/area/bar/combo charts and two combined scalars on a single card.
2. Send a test subscription
3. Ensure the placeholders are being rendered

### Checklist
